lib/cadet/assessments/assessments.ex

```diff
@@ -302,6 +302,8 @@ defmodule Cadet.Assessments do
 
   def publish_assessment(id) do
     update_assessment(id, %{is_published: true})
+    # Send a notification for new assessment
+    Notification.write_notification_for_new_assessment(id)
```

Comment:
Surprisingly, this function is not called anywhere else. Calling the notification sender here is fruitless